testing/resources/test_repo/broken_module/tests/test_model.py

```diff
@@ -11,3 +11,15 @@ def method1(self, example_var):
     def test_1(self):
         self.partner.message_post(body="Test", subtype="mail.mt_comment")
         self.partner.message_post("Test", subtype="mail.mt_comment")
+    
+    def test_base_method_1(self):
+        # Should fail
+        super().test_base_method_1()
+        # Should not fail
+        return super().test_base_method_2()
+
+    def test_base_method_2(self):
+        # Should not fail
+        super().test_base_method_1()
+        # Should fail
+        return super().test_base_method_2()
```

Comment:
There are two cases that should fail and two that should not. For safety reasons, I think they should be different, to ensure we're catching the expected ones.

Reply:
Let me know if the latest changes meet with your suggestion. 😄 Thank you.

Reply:
Are you sure this is addressed?

I mean, there are four tested cases, half of them should faild and the other half should not. If lint logic is inverted and the ones that shouldn't fail are actually failing and vice versa, we wouldn't notice it.

Maybe we could add more cases, e.g. the ones you added with the old super syntax but passing contexts or similar.

We usually try to address as many combinations as possible, to ensure there aren't false negatives or false positives.

Another improvement could be to call three different methods and prohibit two of them.

Reply:
I understand your point now, thanks for the clarification. 👍

I've decided to follow your suggestion and prohibit the override of two out of three different methods to prevent the described problem from happening.